bin/localizely_download.dart

```diff
@@ -30,6 +30,11 @@ Future<void> main(List<String> arguments) async {
     'api-token',
     help: 'Localizely API token.',
   );
+  argParser.addOption(
+    'arb-dir',
+    help: 'Directory of the arb files.',
+    defaultsTo: defaultArbDir,
```

Comment:
Just one thing regarding default value here...

Currently, we are trying to fall back to the values from the `flutter_intl` config from the `pubspec.yaml` file before using default values, in case the argument is not provided (see handling of the `project-id` arg). 

The reason for this is that Flutter Intl IDE plugins use `intl_utils` for generating, uploading, and downloading translations in that way. 

We should fall back something like this before downloading arb files...
```
var arbDir = argResults['arb-dir'] as String;
...
var pubspecConfig = PubspecConfig();
...
arbDir ??= pubspecConfig.arbDir ?? defaultArbDir;

await LocalizelyService.download(projectId, apiToken, arbDir);
```